src/Microsoft.AspNetCore.Rewrite/Internal/ModRewrite/ConditionPatternParser.cs

```diff
@@ -139,7 +138,7 @@ public static ParsedModRewriteInput ParseActionCondition(string condition)
         /// <param name="context"></param>
         /// <param name="invert"></param>
         /// <returns></returns>
-        public static ParsedModRewriteInput ParseProperty(ParserContext context, bool invert)
+        private ParsedModRewriteInput ParseProperty(ParserContext context, bool invert)
```

Comment:
This is correct an instance can be collected while a static class's fields won't. But this shouldn't be related to static methods.